test/java/testing-matchers/update-pom.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
```

Comment:
nit: a short comment about this script adding dependencies for java tests would be nice here as well 👍